packages/wallet/src/services/ProviderTracker/HandleProvider.ts

```diff
@@ -0,0 +1,45 @@
+import { HandleProvider, HandleResolution, ResolveHandlesArgs } from '@cardano-sdk/core';
+import { HealthCheckResponse, NetworkInfoProvider, ProviderError, ProviderFailure } from '@cardano-sdk/core/src';
```

Comment:
Imports from other SDK packages should be referencing the index. 

Also please hoist this file to `src/services`, as ProviderTracker has special meaning that doesn't match what this class does (used for SingleAddressWallet observable implementations)
```suggestion
import { HandleProvider, HandleResolution, ResolveHandlesArgs, HealthCheckResponse, NetworkInfoProvider, ProviderError, ProviderFailure } from '@cardano-sdk/core';
```

Reply:
This was moved in the end to `cardano-services` [f293dc8313](https://github.com/input-output-hk/cardano-js-sdk/pull/732/commits/f293dc8313a0c07bc570e18d025412315abc45ef) should solve this